install_common_crypto.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+COMMON_CRYPTO_DIR="${SDKROOT}/usr/include/CommonCrypto"
+if [ -f "${COMMON_CRYPTO_DIR}/module.modulemap" ]; then
+    exit 0
+fi
+
 FRAMEWORK_DIR="${BUILT_PRODUCTS_DIR}/CommonCrypto.framework"
 
 # Check if module map exists (Xcode 10). If not, generate a dummy module.
```

Comment:
This comment should now be before the previous `if` 😝 , and here it could be another one like:

`# Check if the dummy module has already been created. If not, create it.`